.github/workflows/ci.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         with:
           name: code-coverage-ubuntu-latest-20.x
           path: coverage/
-      - uses: SonarSource/sonarcloud-github-action@v1.9
+      - uses: SonarSource/sonarcloud-github-action@v2
```

Comment:
the last run said it couldnt find this v2 version.

Reply:
My bad, Github action don't support semantic versioning, it's just that good-mannered people publish a tag with the major version, which I forgot about ^^